spec/system/school/courses/target_details_editor_spec.rb

```diff
@@ -187,4 +188,210 @@
     expect(target.prerequisite_targets.count).to eq(1)
     expect(target.prerequisite_targets.first).to eq(target_2_l2)
   end
+
+  context 'admin adds checklist to a target' do
```

Comment:
This doesn't sound like a _context_. Perhaps `when targets have an existing checklist`?